src/components/Articles/VoteButton.jsx

```diff
@@ -0,0 +1,40 @@
+import React, { useState } from "react"
+import axiosInstance from "../utilities/axios"
+
+const VoteButton = ({ articleId, votes, setVotes }) => {
+  const [loading, setLoading] = useState(false)
+
+  const handleVote = async (increment) => {
+    try {
+      setLoading(true)
+
+      // optimistic
+      setVotes((prevVotes) => prevVotes + (increment ? 1 : -1))
+
+      //   await axiosInstance.patch(`/articles/${articleId}`, { increment })
```

Comment:
If this isn't working it's likely due to the request body - have a look at your tests for this endpoint in your backend project. What does the request body look like? I think it takes the vote increment on a key of `inc_votes`
Also ideally your API requests would be separated into their own API file